### PR TITLE
ci: initial commit for gmbuilder

### PR DESCRIPTION
**Pull request by S1nus**
forked a specific release that works with the tutorial, because cosmos-sdk has no main branch.
- gmbuilder-docker-compose.yml starts the gmbuilder.Dockerfile which builds and runs the tutorial alongside a Celestia local devnet 
- `gm-tester.sh` polls the rollup for block #3 (could add other behavior, perhaps testing transactions)
- `.github/workflows/test-tutorial.yml` automates the test

### CODE REVIEW

File: gm-tester.sh [hunk 1]
```diff
@@ -0,0 +1,39 @@
+URL=localhost:26657/block\?height=3
```
Comment (tty47) [inline gm-tester.sh:2]: as a good practice, please add the header to the top of the file:
`sh` 
```suggestion
#!/bin/sh
URL=localhost:26657/block\?height=3
```

File: gmbuilder.Dockerfile [hunk 2]
```diff
@@ -0,0 +1,9 @@
+FROM golang:1.20.3
+
+WORKDIR /
+
+ADD ./ /cosmos-sdk/
+
+COPY script.sh /script.sh
```
Comment (tty47) [inline gmbuilder.Dockerfile]: you can add the permission directly in the copy and skip/remove the `chmod` layer (line 8)

```suggestion
COPY --chmod=+x script.sh /script.sh
```
or:
```suggestion
COPY --chmod=0744 script.sh /script.sh
```

*please, try to build it locally first*

**Review by tty47 — APPROVED** [review]
LGTM! 🚀

**Review by MSevey — COMMENTED** [review]
lgtm, just need to fix the failing lint

**Review by MSevey — APPROVED** [review]

**Pull request merged**